scripts/__init__.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+try:
+    import coloredlogs
```

Comment:
seems to be unused

Reply:
https://github.com/DisnakeDev/disnake/pull/708/commits/f0e26b9847309c7183bbfa9a9c21707bacd83471